Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -33,14 +31,13 @@ open class Container: UIBaseObject {
         }
     }
 
-    public init(loader: Loader = Loader(), options: Options = [:]) {
+    public init(options: Options = [:]) {
         Logger.logDebug("loading with \(options)", scope: "\(type(of: self))")
         self.options = options
-        self.loader = loader
         super.init(frame: CGRect.zero)
         self.sharedData.container = self
         backgroundColor = UIColor.clear
-        loadPlugins()
+        Loader.shared.loadPlugins(in: self)
```

Comment:
Just a suggestion for a later change. Perhaps we should isolate the responsibility of instantiating the plugins in another component, like a factory or factory method, and make the component initialization cleaner.